### DISCUSSION TITLE
Way to force a new markdown cell in Jupyter output?

### DISCUSSION THREAD

**Discussion by jlperla** [quarto-dev/quarto-cli#4288] [Q&A] [answered]
I am writing a `qmd` file and generating an `ipynb` with
```bash
quarto render my_file.qmd --to ipynb
```

Is there any hack that I can force a new markdown cell?  In particular, I have something like
```md
# Question 1
Some math question...

$$
X = ?
$$
(double-click to modify answer)

# Question 2
lots more stuff
```
What I want is a break before the `Question 2` section so that when they doubleclick to type in their answer they don't get a huge cell.

**Comment by cderv** [comment, marked as answer]
This would be the way I think 

````markdown
::: {.cell .markdown}
# Question 1
Some math question...

$$
X = ?
$$
:::

::: {.cell .markdown}
# Question 2
lots more stuff
:::
````

Note that this is a Pandoc feature (https://pandoc.org/MANUAL.html#jupyter-notebooks)  and this would also work with Pandoc only, without Quarto - Quarto allows to render to all Pandoc supported formats (https://quarto.org/docs/output-formats/all-formats.html) 

Some advanced features may not work though for ipynb output

**Comment by jlperla** [comment]
Thanks.  Works beautifully
